docs/SPEC.md

```
@@ -67,17 +67,46 @@ Code block info string tags are matched case-insensitively. `BASH`, `Bash`, and
+- review-structural-integrity.runbook.md
+```
+
+```markdown
+## 2. Review the plan
+### 2.1
+- review-technical-accuracy.runbook.md
+### 2.2
+- review-structural-integrity.runbook.md
+```
+
+### 3.4 Runtime Target Identity
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)